lamindb/migrations/0138_remove_record_linked_users_user_linked_in_records.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.8 on 2025-10-23 16:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("lamindb", "0137_squashed"),
```

Comment:
This cannot depend on the squashed migration, it has to depend on the actual 0137 migration.

Please also rename 0137_squashed to 0138_squashed and append these two lines at the bottom.

Reply:
Also, please add the 0138 migration here: https://github.com/laminlabs/lamindb/blob/b4e4f60481a5b618def4eab2e21ba71f30772ec5/lamindb/migrations/0137_squashed.py#L335